mmaction/datasets/pipelines/loading.py

```diff
@@ -178,6 +178,61 @@ def __call__(self, results):
         return results
 
 
+@PIPELINES.register_module()
+class UntrimSampleFrames(object):
+    """Sample frames from the untrimmed video.
+
+    Required keys are "filename", added or modified keys are "total_frames",
+    "frame_inds", "frame_interval" and "num_clips".
+
+    Args:
+        clip_len (int): The length of sampled clips. Default: 1. (5 for Flow)
```

Comment:
what does `(5 for Flow)` mean? does the default changes to 5 when the input is flow?

Reply:
No, it means when we use UntrimSampleFrames to sample flow from untrimmed videos, we need to set it 5. Since we have wrote it in the config file, I decided to remove it to reduce ambiguity.